subtests/docker_cli/run/run.py

```diff
@@ -46,7 +46,14 @@ def init_subargs(self):
         fqin = self.sub_stuff['fqin']
         subargs.append(fqin)
         subargs += get_as_list(self.config['bash_cmd'])
-        subargs.append(self.config['cmd'])
+        install_cmd = self.config['install_cmd'].strip()
+        if install_cmd:
+            subargs.append("'%s && %s'" % (install_cmd, self.config['cmd']))
+            if 'bash' not in subargs and '-c' not in subargs:
+                self.logwarning("Using install_cmd && cmd w/o bash -c could "
+                                "lead to cancer: %s", subargs)
+        else:
+            subargs.append("'%s'" % self.config['cmd'])
```

Comment:
Here, just add quotes how they're needed.  Also added a comment in config, mentioning that single-quotes will be used.